tests/usbus_cdc_ecm/Makefile

```diff
@@ -9,4 +9,29 @@ USEMODULE += shell
 USEMODULE += shell_cmds_default
 USEMODULE += ps
 
+# Boards that don't have enough endpoints to use CDC ACM together with CDC ECM
+ifeq (,$(filter stdio_%,$(filter-out stdio_cdc_acm,$(USEMODULE))))
+  BOARD_BLACKLIST += \
+    stm32f4discovery \
+    weact-f401cc \
+    weact-f401ce \
+    weact-f411ce \
+    #
+endif
+
+define _usbus_cdc_ecm_blacklist_info
+Warning:
+    The board is blacklisted because it uses `stdio_cdc_acm` as STDIO, which needs
+    a CDC ACM interface in addition to the CDC ECM interface, but the number of
+    available endpoints is not sufficient for this. To use this application you
+    have to use `stdio_uart` or any other `stdio*` module, for example:
+
+    USEMODULE=stdio_uart BOARD=stm32f4discovery make -C tests/usbus_cdc_ecm
```

Comment:
does this work?
```suggestion
    USEMODULE=stdio_uart BOARD=$(BOARD) make -C tests/usbus_cdc_ecm
```

Reply:
Yup I can confirm that this work.
@gschorcht feels free to squash this change whenever you want.

Reply:
Ok, I changed and squashed it directly.